src/utils/clipboard.spec.ts

```diff
@@ -0,0 +1,33 @@
+import Clipboard from '@react-native-clipboard/clipboard';
+import Toast from 'react-native-toast-message';
+
+import { ToastTypeEnum } from '../enums/toast-type.enum';
+import { copyStringToClipboard } from './clipboard.utils';
+
+describe('copyStringToClipboard', () => {
+  const setString = Clipboard.setString as jest.Mock;
+  const showToast = Toast.show as jest.Mock;
```

Comment:
instead of doing these assertions everywhere create a variable with your mock and check if its methods where called.  
example here https://github.com/madfish-solutions/templewallet-mobile/pull/199/files
mockReactNativeQuickActions & mockDeviceEventEmitter